docs/framework/guides/contributing/package-metadata.md

```diff
@@ -0,0 +1,35 @@
+---
+category: framework-contributing
+order: 40
+---
+
+# Package metadata
+
+The package metadata should be located in the `ckeditor5-metadata.json` file in the package root directory. The purpose of this file is to simplify the integration and usage of plugins in CKEditor 5, because this file contains the basic information about plugins that are exported by the package. Only the main ones that put the given functionality together should be described in `ckeditor5-metadata.json`. For example, the [metadata file](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-mention/ckeditor5-metadata.json) in the `ckeditor5-mention` package does not contain the `MentionEditing` and `MentionUI` plugins, but instead it only contains the metadata for the main `Mention` plugin, which is a "glue plugin", that loads the editing and UI features.
+
+## Data format for plugin metadata
+
+The `ckeditor5-metadata.json` file is a JSON object that holds the `plugins` array. Each element of this array should be an object containing information about a different plugin delivered by the package. Below is a list of all matadata properties that can be used to describe the plugin:
+
+* `name` &ndash; A human-readable name of the plugin.
+* `description` &ndash; A human-readable short description of the plugin.
+* `docs` &ndash; An absolute or relative URL for the plugin's documentation. If this URL is relative, it leads to the CKEditor 5 documentation in [https://ckeditor.com/docs/ckeditor5/latest/](https://ckeditor.com/docs/ckeditor5/latest/).
+* `path` &ndash; A path to the file, relative to the metadata file that exports the plugin.
+* `className` &ndash; The name of the class used to create the plugin.
+* `requires` &ndash; An array of the plugin's soft requirements and other non-explicit requirements.
```

Comment:
\+ It should contain class names of plugins that should be included if this plugin is added.

Reply:
Perhaps it would be safer to add support for `repository:pluginName` in the future.

Reply:
Done.